File: app/src/main/java/com/example/vnews/fragmentJ/FragmentKhampha.java
package com.example.vnews.fragmentJ;

import androidx.fragment.app.Fragment;
import android.os.Bundle;
import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;

import com.example.vnews.R;

public class FragmentKhampha extends Fragment {
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        Log.d("asd", "khampha");
        View view = inflater.inflate(R.layout.fragment_khampha, container, false);
        return view;
    }
}

File: app/src/main/java/com/example/vnews/Fragment/Fragment_khampha.kt
package com.example.vnews.Fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.vnews.R

class Fragment_khampha: Fragment() {

    override fun onCreateView(inflater : LayoutInflater, container : ViewGroup?, savedInstanceState : Bundle?) : View {
        val view = inflater.inflate(R.layout.fragment_khampha, container, false)
        return view
    }
}